Gems/ROS2/Code/Source/RobotImporter/Utils/RobotImporterUtils.h

```diff
@@ -138,9 +151,10 @@ namespace ROS2::Utils
     //! Returns the SDF model object which contains the specified joint
     //! @param root - reference to SDF Root object representing the root of the parsed SDF xml document
     //! @param jointName - Name of SDF joint to lookup in the SDF document
+    //! A fully qualified can be of the form "modelname1::nested_modelname1::jointname"
     const sdf::Model* GetModelContainingJoint(const sdf::Root& root, AZStd::string_view jointName);
     //! @param root - reference to SDF Root object representing the root of the parsed SDF xml document
-    //! @param jointName - Name of SDF joint to lookup in the SDF document
+    //! @param joint- SDF joint pointer to lookup in the SDF document
```

Comment:
Technically we're not supposed to have a '-' here for DOxygen, but for consistency with the parameter above it there should be a space before and after the dash.
```suggestion
    //! @param joint - SDF joint pointer to lookup in the SDF document
```

Reply:
I can remove the dashes from all parameters